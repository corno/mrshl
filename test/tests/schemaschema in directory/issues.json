[
    [ "ignoring internal schema", "warning", 1, 1, 1, 1]
]