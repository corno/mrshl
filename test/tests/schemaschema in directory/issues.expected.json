[
	[
		"schema retrieval",
		"warning",
		"found both external and internal schema. ignoring internal schema",
		null,
		null,
		null,
		null
	]
]